src/library_gl.js

```diff
@@ -7611,6 +7611,7 @@ var LibraryGL = {
       cb.stride = stride;
       cb.ptr = ptr;
       cb.clientside = true;
+      cb.vertexAttribPointer = GLctx.vertexAttribPointer;
```

Comment:
this is actually risky. I'm not sure what happens if the context is destroyed or replaced, but I strongly suspect a reference to a function on it would be invalid. Instead, creating a function like in the case lower down is guaranteed to be safe.

Reply:
First I'd like to point out that `GLctx.vertexAttribPointer` is `GLctx.__proto__.vertexAttribPointer`, which is either `WebGLRenderingContext.prototype.vertexAttribPointer` or `WebGL2RenderingContext.prototype.vertexAttribPointer`, so `cb.vertexAttribPointer` always refers to the same function object.
Then what if `GLctx` is replaced or destroyed? Maybe it would result in *undefined behavior*. A sane application should be responsible for setting up the vertex arrays properly when it creates a new context, and draw functions should never be called after the context is destroyed.
I still made the requested change though because it seems to be more eye-candy.